test/unit/pipeline/FetchUnitTest.cc

```diff
@@ -207,7 +220,8 @@ TEST_F(PipelineFetchUnitTest, halted) {
       .WillByDefault(DoAll(SetArgReferee<3>(mOp), Return(4)));
```

Comment:
Similar commnets apply here regarding predecode return value and EXPECT_CALL times expected